sys/include/net/gnrc/netif/internal.h

```diff
@@ -346,14 +359,20 @@ static inline bool gnrc_netif_is_rtr_adv(const gnrc_netif_t *netif)
  * @return  true, if the interface represents a 6LN
  * @return  false, if the interface does not represent a 6LN
  */
-#if ((GNRC_NETIF_NUMOF > 1) && defined(MODULE_GNRC_SIXLOWPAN)) || \
-    defined(MODULE_GNRC_SIXLOENC) || defined(DOXYGEN)
-bool gnrc_netif_is_6lo(const gnrc_netif_t *netif);
-#elif (GNRC_NETIF_NUMOF == 1) && defined(MODULE_GNRC_SIXLOWPAN)
-#define gnrc_netif_is_6lo(netif)                (true)
-#else
-#define gnrc_netif_is_6lo(netif)                (false)
-#endif
+static inline bool gnrc_netif_is_6lo(const gnrc_netif_t *netif)
+{
+    if((!gnrc_netif_highlander() &&
```

Comment:
[Vera++](https://travis-ci.org/github/RIOT-OS/RIOT/builds/660563543#L224) pointed out an error here.

```suggestion
    if ((!gnrc_netif_highlander() &&
```